chia/full_node/full_node.py

```diff
@@ -1097,10 +1104,6 @@ async def sync_from_fork_point(
         summaries: list[SubEpochSummary],
     ) -> None:
         self.log.info(f"Start syncing from fork point at {fork_point_height} up to {target_peak_sb_height}")
-        peers_with_peak: list[WSChiaConnection] = self.get_peers_with_peak(peak_hash)
-        fork_point_height = await check_fork_next_block(
-            self.blockchain, fork_point_height, peers_with_peak, node_next_block_check
-        )
```

Comment:
we call finish_sync in the finally clause even if we fail sync_from_fork_point, i needed the fork_point outside of sync_from_fork_point so i can pass it to finish_sync even if we throw while syncing, we can talk about whether we want to call finish_sync in case of a failure but it felt like its outside the scope of the pr